src/app/app.component.ts

```diff
@@ -11,15 +11,19 @@ export class AppComponent {
   fhirPreview: string;
   questionnaire = 'bmi';
   fhir = fhir;
-  linkId = context;
+  phq9ScoreLinkId = phq9ScoreLinkId;
 
   constructor(private ruleEditorService: RuleEditorService) { }
-
+s
   onChange(): void {
     this.fhirPreview = '';
   }
 
   onSave(fhirResult): void {
     this.fhirPreview = JSON.stringify(fhirResult, null, 2);
   }
+
+  addTotalScoreRule(): void {
+    const result = this.ruleEditorService.addTotalScoreRule(fhir.phq9, phq9ScoreLinkId);
```

Comment:
What are you doing with the result? You don't really have to add this function here, you can just keep the unit test for the functionality.

Reply:
I think it was to get the Calculate Total Scores button to work. Should I set it equal to this.fhirPreview instead?

Reply:
I don't think we need the button on the UI.

Reply:
Fixed, I commented it out but can also just remove it if you'd like.

Reply:
Yes, please remove don't comment it out. It's available in git history if we need it back.